dashboard/modules/metrics/dashboards/default_dashboard_panels.py

```diff
@@ -380,6 +380,90 @@ def max_plus_pending(max_resource, pending_resource):
         fill=0,
         stack=False,
     ),
+    Panel(
```

Comment:
Hmm we should probably build a data panels like we did for ray serve? (serve_dashboard_panels.py)

Reply:
I think doing a separate json for data makes sense but it complicates things the UI more. Maybe for ray 2.8, we keep things in the default dashboard.

For example, in the UI we have a "view in grafana" button in the metrics page. That button will not lead to the data metrics since that is a different grafana dashboard.

If we put data in a separate section at the bottom, we may need to add a different "view in grafana" button next to the data heading.

Reply:
Right now data metrics is in grouped with the default metrics in both ray dashboard and grafana dashboard.

Reply:
it probably makes sense to do a separate grafana dashboard for datasets because that follows the same pattern as the serve dashboard.

We should probably rename the "default dashboard" to "core dashboard" but that can be done later.


Even if we make it a separate dashboard, we can still embed it into the metrics page using a different URL. I think that's a good short term solution since we can later move embedded metrics out of the UI easily (since it's unlikely any system depends on the Dashboard UI) but we can't easily remove metrics from the grafana dashboard, since there may be some systems that depend on the json output. (Anyscale does for example)